src/app/components/MessageBanner/index.tsx

```diff
@@ -0,0 +1,82 @@
+/** @jsx jsx */
+/* @jsxFrag React.Fragment */
+import React, { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import Paragraph from '../Paragraph';
+import Heading from '../Heading';
+import Image from '../Image';
+import Text from '../Text';
+import { LeftChevron, RightChevron } from '../icons';
+import styles from './index.styles';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+interface MessageBanner {
+  summaries: {
+    imageUrl?: string;
+    link: string;
+    imageAlt: string;
```

Comment:
This should probably be optional (we set it to empty string anyway)
```suggestion
    imageAlt?: string;
```

Reply:
Cool committing!